<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(28, 28, 28);
font: 13pt &quot;Helvetica&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>241</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">padding-left: 15px;
padding-right: 15px;</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-167</y>
       <width>209</width>
       <height>456</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="itemName">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(126, 124, 127);
border-radius: 3%;</string>
        </property>
        <property name="text">
         <string>Item</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="itemValue">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(227, 224, 229);
border-radius: 3%;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="itemName2">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(126, 124, 127);
border-radius: 3%;</string>
        </property>
        <property name="text">
         <string>Item</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="itemValue2">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(227, 224, 229);
border-radius: 3%;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="itemName3">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(126, 124, 127);
border-color: rgba(255, 255, 255, 0);
border-radius: 3%;</string>
        </property>
        <property name="text">
         <string>Item</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="itemValue3">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(227, 224, 229);
border-radius: 3%;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="itemName4">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(126, 124, 127);
border-radius: 3%;</string>
        </property>
        <property name="text">
         <string>Item</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="itemValue4">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(227, 224, 229);
border-radius: 3%;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLineEdit" name="itemName5">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(126, 124, 127);
border-radius: 3%;</string>
        </property>
        <property name="text">
         <string>Item</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="itemValue5">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(227, 224, 229);
border-radius: 3%;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLineEdit" name="itemName6">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(126, 124, 127);
border-color: rgba(255, 255, 255, 0);
border-radius: 3%;
</string>
        </property>
        <property name="text">
         <string>Item</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="itemValue6">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(227, 224, 229);
border-radius: 3%;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLineEdit" name="itemName7">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(126, 124, 127);
border-radius: 3%;</string>
        </property>
        <property name="text">
         <string>Item</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="itemValue7">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(227, 224, 229);
border-radius: 3%;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLineEdit" name="itemName8">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(126, 124, 127);
border-radius: 3%;</string>
        </property>
        <property name="text">
         <string>Item</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="itemValue8">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(227, 224, 229);
border-radius: 3%;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLineEdit" name="itemName9">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(126, 124, 127);
border-color: rgba(255, 255, 255, 0);
border-radius: 3%;</string>
        </property>
        <property name="text">
         <string>Item</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="itemValue9">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(227, 224, 229);
border-radius: 3%;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLineEdit" name="itemName10">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(126, 124, 127);
border-radius: 3%;</string>
        </property>
        <property name="text">
         <string>Item</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="itemValue10">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(227, 224, 229);
border-radius: 3%;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLineEdit" name="itemName11">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(126, 124, 127);
border-radius: 3%;</string>
        </property>
        <property name="text">
         <string>Item</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLineEdit" name="itemValue11">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(227, 224, 229);
border-radius: 3%;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLineEdit" name="itemName12">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(126, 124, 127);
border-color: rgba(255, 255, 255, 0);
border-radius: 3%;</string>
        </property>
        <property name="text">
         <string>Item</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QLineEdit" name="itemValue12">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(227, 224, 229);
border-radius: 3%;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLineEdit" name="itemName13">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(126, 124, 127);
border-color: rgba(255, 255, 255, 0);
border-radius: 3%;
</string>
        </property>
        <property name="text">
         <string>Item</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QLineEdit" name="itemValue13">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(227, 224, 229);
border-radius: 3%;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLineEdit" name="itemName14">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(126, 124, 127);
border-radius: 3%;</string>
        </property>
        <property name="text">
         <string>Item</string>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QLineEdit" name="itemValue14">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(227, 224, 229);
border-radius: 3%;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="14" column="0">
       <widget class="QLineEdit" name="itemName15">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(126, 124, 127);
border-radius: 3%;</string>
        </property>
        <property name="text">
         <string>Item</string>
        </property>
       </widget>
      </item>
      <item row="14" column="1">
       <widget class="QLineEdit" name="itemValue15">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(227, 224, 229);
border-radius: 3%;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="15" column="0">
       <widget class="QLineEdit" name="itemName16">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(126, 124, 127);
border-radius: 3%;
</string>
        </property>
        <property name="text">
         <string>Item</string>
        </property>
       </widget>
      </item>
      <item row="15" column="1">
       <widget class="QLineEdit" name="itemValue16">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(227, 224, 229);
border-radius: 3%;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item row="16" column="0">
       <widget class="QLineEdit" name="itemName17">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(126, 124, 127);
border-radius: 3%;
</string>
        </property>
        <property name="text">
         <string>Item</string>
        </property>
       </widget>
      </item>
      <item row="16" column="1">
       <widget class="QLineEdit" name="itemValue17">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(227, 224, 229);
border-radius: 3%;</string>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 30pt &quot;.AppleSystemUIFont&quot;;</string>
    </property>
    <property name="text">
     <string>Salary:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="salaryValue">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>113</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(222, 219, 225);
border-radius: 6%;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>240</y>
      <width>231</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 36pt &quot;Helvetica&quot;;</string>
    </property>
    <property name="text">
     <string>Budget Total:</string>
    </property>
   </widget>
   <widget class="QLabel" name="total">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>240</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 40pt &quot;.AppleSystemUIFont&quot;;
color: #00ff00;</string>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="net">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>320</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 40pt &quot;.AppleSystemUIFont&quot;;
color: #00ff00;</string>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>320</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 30pt &quot;Helvetica&quot;;</string>
    </property>
    <property name="text">
     <string>Budget vs Salary:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(212, 210, 215);
border-radius: 6%;</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>20</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(212, 210, 215);
border-radius: 6%;</string>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>749</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
